### DISCUSSION TITLE
Seasonals

### DISCUSSION THREAD

**Discussion by AxeldeWater** [sdaqo/anipy-cli#202] [Q&A] [answered]
So I don't use "Seasonals" and I don't really understand what it is about. What is it? Why would somebody use it? How do you use it? How does it improve the experience?

**Comment by sdaqo** [comment, marked as answer]
Well, because a lot of people watch seasonal anime, in other words "new anime" that air a new episode every week. A lot of people like discussing or watching discussions on anime and those normally happen while the anime is airing. The mode makes watching those convenient because you can add them at the start of a new season (usually spring, summer, fall, winter) and batch download/watch the newest episodes available without having to keep a schedule and check e.g. livechart every day: https://www.livechart.me/fall-2024/tv

**Reply by AxeldeWater** [reply]
Aha, interesting. Do I understand correctly that when you go into seasonal mode, it will show you the newest episodes available that you haven't watched yet from the series you put in your seasonals?

**Reply by sdaqo** [reply]
Yup, exactly.

**Reply by sdaqo** [reply]
It's basically: look whats airing this season, decide what you wanna watch, put it into the Seasonal modes via the 'a' (add) command and check back every other day to watch the newest stuff you have not watched. The mode keeps track of your last watched/downloaded.

**Reply by sdaqo** [reply]
Another thing: you can also use the MAL mode for that, it can basically do the same thing but it syncs to your MAL account.

**Reply by AxeldeWater** [reply]
Does this also work for series that don't air anymore? Because what I do at the moment is 1. watch episodes 2. When I'm done, I go to the next episode and immediately quit 3. When I want to continue watching, I go to history mode to continue with the episode where I was when I quit the program. 

I found it pretty annoying to have to skip to the next episode just so that I could continue from that episode immediately next time. Would seasonals fix this annoyance with non-airing series too?

**Reply by sdaqo** [reply]
Yes you can use it for that,
1. Add anime
2. 'w' command to binge watch and choose the anime you wanna watch (just press enter while the anime is selected)
3. watch as much as you want
4. if you are done close the player (or let the episode play out, the player should close automatically)
5. ctrl+c anipy-cli

This should work for your usecase

You can also keep track of your anime like that.

**Reply by sdaqo** [reply]
Well and if you wanna have it fancier you can use the MAL mode, this would sync accross devices.
